assets/javascript/app.js

```diff
@@ -0,0 +1,19 @@
+window.onload = function() {
+
+};
+
+
+
+var intervalId;
+var clockRunning = false;
+
+$("timer").text("00:00");
+
```

Comment:
Set the timer as a var with a value of the amount of time you want to start with.